music/src/core/compat/navigator_node.ts

```diff
@@ -18,4 +18,4 @@
  * limitations under the License.
  */
 
-export const fetch = window.fetch;
+export const userAgent = '';
```

Comment:
lolsob

Reply:
(actually, is this used anywhere?)

Reply:
Yeah, it is :( (https://github.com/magenta/magenta-js/blob/master/music/src/coconet/coconet_utils.ts#L27)

Reply:
brilliant